### ISSUE TITLE
llvm-cov and nextest arguments in "inverted" positions

### ISSUE THREAD

**Issue by azriel91** [taiki-e/cargo-llvm-cov#151]
Heya, I'm trying `cargo-llvm-cov 0.3.0` with `nextest` in a workspace, and it appears that:

* arguments for the `cargo llvm-cov` application need to be after the `nextest` subcommand, and
* arguments for `nextest` need to be passed before `nextest`

Example:

```bash
nextest_args=(--workspace)
llvm_cov_args=(--workspace --open --output-dir ./target/coverage)

# Passes llvm_cov_args through to nextest, doesn't do what is expected
# This is how I would expect to write the command
cargo llvm-cov $llvm_cov_args[@] nextest $nextest_args[@]

# This does what is expected
cargo llvm-cov $nextest_args[@] nextest $llvm_cov_args[@]
```

I haven't managed to figure out whether it's by design by `clap`.

**Comment by taiki-e** [comment]
Yeah, support for passthrough arguments has some bugs: see also https://github.com/taiki-e/cargo-llvm-cov/pull/144#issuecomment-1072750780 and https://github.com/taiki-e/cargo-llvm-cov/pull/144#issuecomment-1072772281

The proposed (and accepted) fix is https://github.com/taiki-e/cargo-llvm-cov/pull/144#issuecomment-1072955602, but has not yet been implemented.

**Comment by taiki-e** [comment]
The fix has been published in [0.5.0](https://github.com/taiki-e/cargo-llvm-cov/releases/tag/v0.5.0)
